docker/install_dependencies.sh

```diff
@@ -34,33 +34,15 @@ sudo apt-get install -y --allow-unauthenticated gcc-7 g++-7 python3.6 unzip tar
   build-essential pkg-config lbzip2 libprotobuf-dev protobuf-compiler patch rsync \
   wget vim nano xterm libprotobuf-dev git
 sudo rm -rf /var/lib/apt/lists/*
-export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
 
-#update-alternatives --set python /usr/bin/python3.7
 sudo apt-get clean
 sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 20 --slave /usr/bin/g++ g++ /usr/bin/g++-7
 sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
 sudo update-alternatives --set python /usr/bin/python3.7
 
-cd /home/carma/src
-
-# Install Protobuf - OPTIONAL
-#
-# Pulled in via apt-get instead of compiled
-#
-#wget "https://github.com/protocolbuffers/protobuf/archive/refs/tags/v3.3.0.tar.gz"
-#tar xvf v3.3.0.tar.gz
-#cd protobuf-3.3.0
-#./autogen.sh
-#./configure --prefix=/usr
-
-#make
-#make check
-#sudo make install
-
 # Install SUMO-1.15.0
 cd /home/carma/src/
-wget "https://github.com/eclipse/sumo/archive/refs/tags/v1_15_0.tar.gz"
+wget -q "https://github.com/eclipse/sumo/archive/refs/tags/v1_15_0.tar.gz"
 sudo mkdir -p /opt/sumo
 sudo chown -R carma:carma /opt/sumo
 tar xvf v1_15_0.tar.gz -C /opt/sumo
```

Comment:
I suspect similar to CARLA, Sumo Traci client comes packaged with sumo. If that is the case I think we can push any installation changes to sumo into a different PR.

Reply:
Yeah traci python API is included under sumo/tools which I do not think is built or installed by CMake. See guide for including sumo traci API in python (https://sumo.dlr.de/docs/TraCI/Interfacing_TraCI_from_Python.html)

Reply:
Same replies as the previous comment. Are we able to install TraCI through `pip`?

Reply:
We need SUMO either way since CDA Sim launches SUMO. I assume this is just a way of making sure we get the TraCI client that corresponds to our SUMO installation.